### PR TITLE
Prevent crash when instrument does not exist.

### PR DESCRIPTION
**Pull request by UlysseFG**

### CODE REVIEW

File: backend/fms_core/template_importer/row_handlers/sample_pooling/pool.py
Comment (nafiz1001) [inline backend/fms_core/template_importer/row_handlers/sample_pooling/pool.py:1]: Did you just add an `if` and indent a whole paragraph?

File: backend/fms_core/template_importer/row_handlers/sample_submission/pool.py
Comment (nafiz1001) [inline backend/fms_core/template_importer/row_handlers/sample_submission/pool.py:1]: I'm guessing the same could be said about this?

**Review by nafiz1001 — APPROVED** [review]
Those two codes look suspiciously similar; there might be a way to reuse those two codes into a function.
Aside from that, there's nothing wrong that stood out ;)

File: backend/fms_core/template_importer/row_handlers/sample_pooling/pool.py
Comment (UlysseFG) [reply backend/fms_core/template_importer/row_handlers/sample_pooling/pool.py:1]: yeah i did ... The errors was that when failing to find an instrument we would continue as if we actually did find one ... This fixes that.

**Comment by UlysseFG** [comment]
> Those two codes look suspiciously similar; there might be a way to reuse those two codes into a function. Aside from that, there's nothing wrong that stood out ;)

yeah they are pretty similar ... We'll see about this once the code stabilize some and we are confident we can be sure these things will stay ... I do not really want to work too much on refactoring until the lab has used the system a bit and agrees those functionality work as intended.

**Pull request merged**